ci/run.sh

```diff
@@ -16,7 +16,7 @@ fi
 mkdir build && cd build
 conan install .. --build missing --profile release
 cmake ${CMAKE_OPTIONS} -DCMAKE_INSTALL_PREFIX=install ..
-make -j2
+make -j2 VERBOSE=1
```

Comment:
Do travis builders have only two cores? Otherwise we could use `make -j $(nproc)` (not sure if it works on a mac though). However, that's just a minor thing.

Reply:
Exactly only two. I think once I tried to use `make -j` but it was counterproductive:

https://docs.travis-ci.com/user/reference/overview/